proposals/nnnn-actors.md

```diff
@@ -210,13 +210,13 @@ extension BankAccount {
 
 This restriction prevents exclusivity violations where the modification of the actor-isolated `balance` is initiated by passing it as `inout` to a call that is then suspended, and another task executed on the same actor then fails with an exclusivity violation in trying to access `balance` itself.
 
-#### Escaping reference types
+#### Escaping references
```

Comment:
not super sure about changing the section name tbh, in pure text the "thing with value semantics" reads well, but the title here now reads worse to me tbh.

Minor nitpick only though

Reply:
Yeah I was unsure about how best to deal with this title. “Escaping references” isn’t great, but it felt weird to leave the phrasing as “Escaping reference types” when the body of the section has been reworded to specifically *not* discuss things in terms of “reference types.” And other potential phrasings like “Escaping reference semantics” or “Escaping types with reference semantics” didn’t seem quite right either... 🤔 If you think the original title isn’t too misleading I’m happy to leave as-is. Or, if you have any alternative suggestions, I’d love to hear!